libs/ui-kit/toggletip/toggletip.component.scss

```diff
@@ -0,0 +1,50 @@
+cz-toggletip {
+  display: none;
+}
+
+.cz-toggletip-content {
+  --bg-color: var(--cz-color-white);
+  --font-size: 0.875rem;
+  position: relative;
+  max-width: 23rem;
+  padding: 1.5rem;
+  background-color: var(--bg-color);
+  box-shadow: 0 2px 8px 0 rgba(40, 41, 61, 0.04), 0 16px 24px 0 rgba(96, 97, 112, 0.16);
+  font-size: var(--font-size);
+
+  .cz-toggletip-close {
+    position: absolute;
+    top: 0.25rem;
+    right: 0.25rem;
+
+    height: 1.5rem;
```

Comment:
same as the above statement, px is better I believe

Reply:
I changed top and right to px which indeed make more sense, but here since it's  about an ico button, I feel it's better to make it bigger if the user sets a higher font size, as it would make the button easier to click on? What do you think? 🤔

Reply:
if it works fine for you with higher font sizes, we can keep it as is :)